Sources/SwiftDocC/Infrastructure/Extensions/DeclarationFragments+Simplify.swift

```diff
@@ -0,0 +1,78 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import SymbolKit
+
+fileprivate typealias Fragment = SymbolKit.SymbolGraph.Symbol.DeclarationFragments.Fragment
+
+extension [Fragment] {
```

Comment:
Out of curiosity, what made you decide to implement this logic here as opposed to in SymbolKit (such as adding a new property to DeclarationFragments)?

Reply:
Not wanting to split the PR across repos, mainly. 😅 

This logic is pretty specific to the kind of display that DocC is wanting to do, and isn't representative of what data is coming directly from the symbol graph JSON file. I feel like SymbolKit is more of a data-representation library, and adding more custom-logic pieces like this somewhat muddles that.

Reply:
Agreed. This feels like it belongs in DocC.